kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -259,6 +274,15 @@ private[kyuubi] class EngineRef(
     }
   }
 
+  private def removeFailedHandleSpace(): Unit = {
+    val sessionRecordNode = SessionNodeUtil.combine2SessionNode(engineSpace)
+    val closeHandleSpace = s"$sessionRecordNode/$engineRefId"
+    RebalancedSparkEnginePolicy.deleteHandlePath(
+      conf,
+      closeHandleSpace,
+      SpaceBean(getCommonParent, user))
```

Comment:
please avoid using "bean", it's only popular on projects using the EJB / Spring framework.